meta-mender-core/recipes-mender/mender-artifact/mender-artifact_git.bb

```diff
@@ -29,6 +29,8 @@ def mender_branch_from_preferred_version(d):
         # If the preferred version is some kind of version, use the branch name
         # for that one (1.0.x style).
         return match.group(0) + "x"
+    elif version.endswith("-git%"):
```

Comment:
Q: When are we unifying this functionality somewhere?

Reply:
My "unification muscle" twitched a bit as well. I don't promise to do it now, but let me try and see if it causes problems.

Reply:
Bah! There is a SHA in there which is specific to each repository. It *can* be unified, but I'm leaving it alone for now. A task for another day.

Reply:
At least we tried :)